src/Components/CreateImageWizard/utilities/useValidation.tsx

```diff
@@ -389,28 +433,108 @@ export function useUsersValidation(): StepValidation {
+  const sshKeyError = getSshKeyErrorMsg(userSshKey);
+
+  const { isUserNameValidValue, isSshKeyValidValue, isPasswordValidValue } =
+    calculateValidationFlags(userName, userPassword, userSshKey);
+
   const canProceed =
     // Case 1: there is no users
     users.length === 0 ||
-    // Case 2: All fields are empty
-    (userName === '' && userPassword === '' && userSshKey === '') ||
-    // Case 3: userName is valid and SshKey is valid
-    (userName &&
-      isUserNameValid(userName) &&
-      userSshKey &&
-      isSshKeyValid(userSshKey));
+    // Case 2: userName is valid and SshKey or Password is valid
+    (isUserNameValidValue && (isSshKeyValidValue || isPasswordValidValue));
 
   return {
     errors: {
-      userName:
-        userName && !isUserNameValid(userName) ? 'Invalid user name' : '',
-      userSshKey:
-        userSshKey && !isSshKeyValid(userSshKey) ? 'Invalid SSH key' : '',
+      userName: userNameError,
+      userSshKey: sshKeyError,
     },
     disabledNext: !canProceed,
   };
 }
 
+const calculateValidationFlags = (
+  userName: string,
+  userPassword: string,
+  userSshKey: string
+): ValidationFlags => {
+  const isUserNameValidValue = !!userName && isUserNameValid(userName);
+  const isSshKeyValidValue = !!userSshKey && isSshKeyValid(userSshKey);
+  const isPasswordValidValue =
+    !!userPassword && checkPasswordValidity(userPassword).isValid;
+  return { isUserNameValidValue, isSshKeyValidValue, isPasswordValidValue };
+};
+
+export const checkPasswordValidity = (
+  value: string
+): PasswordValidationResult => {
+  if (!value) {
+    return {
+      isValid: false,
+      strength: getStrength(0),
+      validationState: {
+        ruleLength: 'indeterminate',
+        ruleCharacters: 'indeterminate',
+      },
+    };
+  }
+  const isEncrypted = /^\$([^$]+)\$/.test(value);
+  if (isEncrypted) {
+    return {
+      isValid: true,
+      strength: getStrength(0),
+      validationState: {
+        ruleLength: 'success',
+        ruleCharacters: 'success',
+      },
+    };
+  }
+
+  const trimmedValue = value.trim();
+  const isLengthValid = trimmedValue.length >= 6 && trimmedValue.length <= 128;
+  const { rulesCount, strCount } = countCharacterTypes(value);
+
+  const validationState: ValidationState = {
+    ruleLength: isLengthValid ? 'success' : 'error',
+    ruleCharacters: rulesCount >= 3 ? 'success' : 'error',
+  };
+
+  return {
+    isValid: isLengthValid && (rulesCount >= 3 || isLengthValid),
+    strength: getStrength(strCount),
+    validationState: validationState,
+  };
+};
+
+const getStrength = (
+  strCount: number
+): PasswordValidationResult['strength'] => {
+  if (strCount < 3)
+    return { variant: 'error', icon: <ExclamationCircleIcon />, text: 'Weak' };
+  if (strCount < 5)
+    return {
+      variant: 'warning',
+      icon: <ExclamationTriangleIcon />,
+      text: 'Medium',
+    };
+  return { variant: 'success', icon: <CheckCircleIcon />, text: 'Strong' };
+};
```

Comment:
How would I get to see any of these apart from the "Strong"?

![image](https://github.com/user-attachments/assets/41b431e4-1cc1-4ccc-befd-8f55ae120bcd)

Reply:
as discussed in the slack channel, we will see only the Strong label as it shows in https://www.patternfly.org/components/password-strength/
remove other conditions